app/cafe.py

```diff
@@ -0,0 +1,19 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str | None:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("no key == vaccine")
```

Comment:
The error message 'no key == vaccine' is not very descriptive. According to the checklist, you should use descriptive error messages. Consider changing it to something like 'Visitor is not vaccinated: missing vaccine information.'